themes/digital.gov/layouts/services/single.html

```diff
@@ -1,66 +1,57 @@
 {{- define "content" -}}
+  {{ $path := "" }}{{ with .File }}
+    {{ $path = .Path }}
+  {{ else }}
+    {{ $path = .Path }}
+  {{ end }}
   <main role="main" id="main-content">
-    <article>
+    <article class="grid-container grid-container-desktop">
       <header>
```

Comment:
This pattern of `main > article > header` isn't the greatest. We could simplify by removing the article/header. The aside _should_ be outside of `<main>`.

We should take care of this in a future card.

Reply:
Agreed, there is a card in the backlog addressing these markup changes for landing pages.